Description: Minor man pages changes
 Add backslash to hypen to allow grof to interpret minus signs. Add
 .na and .ad requests to avoid "cannot adjust line" man error.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>

---
Origin: other
Bug: http://bugs.schedmd.com/show_bug.cgi?id=1676
Last-Update: 2015-05-19

--- slurm-llnl-14.11.6.orig/doc/man/man1/scontrol.1
+++ slurm-llnl-14.11.6/doc/man/man1/scontrol.1
@@ -779,11 +779,13 @@ The list of nodes allocated to the job.
 The NodeIndices expose the internal indices into the node table
 associated with the node(s) allocated to the job.
 .TP
+.na
 \fINtasksPerN:B:S:C\fP=
 <tasks_per_node>:<tasks_per_baseboard>:<tasks_per_socket>:<tasks_per_core>
 Specifies the number of tasks to be started per hardware component (node,
 baseboard, socket and core).
 Unconstrained values may be shown as "0" or "*".
+.ad
 .TP
 \fIPreemptTime\fP
 Time at which job was signaled that it was selected for preemption.
@@ -796,10 +798,12 @@ Time the job ran prior to last suspend.
 \fIReason\fP
 The reason job is not running: e.g., waiting "Resources".
 .TP
+.na
 \fIReqB:S:C:T\fP=
 <baseboard_count>:<socket_per_baseboard_count>:<core_per_socket_count>:<thread_per_core_count>
 Specifies the count of various hardware components requested by the job.
 Unconstrained values may be shown as "0" or "*".
+.ad
 .TP
 \fISecsPreSuspend\fP=<seconds>
 If the job is suspended, this is the run time accumulated by the job
--- slurm-llnl-14.11.6.orig/doc/man/man5/slurm.conf.5
+++ slurm-llnl-14.11.6/doc/man/man5/slurm.conf.5
@@ -1281,8 +1281,8 @@ May not exceed 65533.
 .TP
 \fBMemLimitEnforce\fR
 If set to "no" then Slurm will not terminate the job or the job step
-if they exceeds the value requested using the --mem-per-cpu option of
-salloc/sbatch/srun. This is useful if jobs need to specify --mem-per-cpu
+if they exceeds the value requested using the \-\-mem\-per\-cpu option of
+salloc/sbatch/srun. This is useful if jobs need to specify \-\-mem\-per\-cpu
 for scheduling but they should not be terminate if they exceed the
 estimated value. The default value is 'yes', terminate the job/step
 if exceed the requested memory.
